starter_code/app.js

```diff
@@ -54,5 +54,13 @@ app.locals.title = 'Express - Generated with IronGenerator';
 const index = require('./routes/index');
 app.use('/', index);
 
+const movies = require('./routes/movies');
+app.use('/', movies);
+
+const movie = require('./routes/movie.js');
+app.use('/', movie);
```

Comment:
As both routes are for the same Model you could have created both at the same routes file (./routes/movies).